external/fv3fit/fv3fit/_shared/pytorch/Graphloss.py

```diff
@@ -0,0 +1,56 @@
+import torch
+import dataclasses
+
+
+def _standard_mse():
+    loss = torch.nn.MSELoss()
+
+    def custom_loss(y_true, y_pred):
+        return loss(y_pred, y_true)
+
+    return custom_loss
+
+
+def _standard_mae():
+    loss = torch.nn.L1Loss()
+
+    def custom_loss(y_true, y_pred):
+        return loss(y_true, y_pred)
+
+    return custom_loss
+
+
+@dataclasses.dataclass
+class LossConfig:
+    """
+    Attributes:
+        loss_type: one of "mse" or "mae"
+        multistep: number of successive loss calculation before each optimization
```

Comment:
What is a successive loss calculation? Do you mean number of chained timesteps to include in optimized loss?

Reply:
I meant the number of multistep losses, which I changed to make it clearer.